Specs/Scene/Instanced3DModel3DTileContentProviderSpec.js

```diff
@@ -0,0 +1,176 @@
+/*global defineSuite*/
+defineSuite([
+        'Scene/Instanced3DModel3DTileContentProvider',
+        'Core/Cartesian3',
+        'Core/Color',
+        'Core/defaultValue',
+        'Core/defined',
+        'Core/HeadingPitchRange',
+        'Core/loadArrayBuffer',
+        'Scene/Cesium3DTileContentState',
+        'Scene/Cesium3DTileset',
+        'Specs/createCanvas',
+        'Specs/createScene',
+        'Specs/pollToPromise'
+    ], function(
+        Instanced3DModel3DTileContentProvider,
+        Cartesian3,
+        Color,
+        defaultValue,
+        defined,
+        HeadingPitchRange,
+        loadArrayBuffer,
+        Cesium3DTileContentState,
+        Cesium3DTileset,
+        createCanvas,
+        createScene,
+        pollToPromise) {
+    "use strict";
+
+    var scene;
+
+    beforeAll(function() {
+        scene = createScene();
+
+        // One instance in each data set is always located in the center, so point the camera there
+        var center = Cartesian3.fromRadians(-1.31995, 0.69871, 5.0);
+        scene.camera.lookAt(center, new HeadingPitchRange(0.0, -1.57, 10.0));
+    });
+
+    afterAll(function() {
+        scene.destroyForSpecs();
+    });
+
+    afterEach(function() {
+        scene.primitives.removeAll();
+    });
+
+    function verifyRender(tileset) {
```

Comment:
As we add more 3D Tiles tests, we may want to move this and friends to a utility class/function(s) in a separate file(s).